docs/source/user_guide/automl.ipynb

```diff
@@ -148,6 +148,47 @@
     "automl_custom = evalml.automl.AutoMLSearch(problem_type='multiclass', allowed_pipelines=[CustomMulticlassClassificationPipeline])"
    ]
   },
+  {
```

Comment:
@bchen1116 could we move this content below "Get best pipeline" and above "Access raw results"?